Sources/Commands/SwiftBuildTool.swift

```diff
@@ -157,6 +162,29 @@ public struct SwiftBuildTool {
             }
         
             switch mode {
+            case .repl(let toolchain):
+                let (rootPackage, externalPackages) = try fetch(opts.path.root)
+                let (modules, externalModules, _) = try transmute(rootPackage, externalPackages: externalPackages)
+
+                /// TODO: Need to create DSOs for C executables as well.
+                let products: [Product] = modules.flatMap{ $0 as? SwiftModule }
+                                                 .map{ Product(name: $0.name, type: .Library(.Dynamic), modules: [$0]) }
+
+                let yaml = try describe(opts, .debug, modules, Set(externalModules), products, toolchain: toolchain)
```

Comment:
I thought about this more, and what I would like to do instead here is ensure that the default `build.yaml` has targets for all of the situations we might want to build for (similar to what we do for tests), but not all of them need to be built by default. So we shouldn't need to regenerate the build file just for this option.

Does that make sense?